src/main/services/window-manager.ts

```diff
@@ -27,12 +30,16 @@ export class WindowManager {
   }
 
   public static createMainWindow() {
-    // Create the browser window.
+    const primaryDisplay = screen.getPrimaryDisplay();
+    const { width, height } = primaryDisplay.workAreaSize;
+
     this.mainWindow = new BrowserWindow({
-      width: 1200,
-      height: 720,
-      minWidth: 1024,
-      minHeight: 540,
+      height: height,
+      width: width,
+      minHeight: WINDOW_MIN_HEIGHT,
+      minWidth: WINDOW_MIN_WIDTH,
+      x: primaryDisplay.bounds.x,
```

Comment:
This is the relevant part: making sure the new window will be placed at the right (x, y) based on the bounds of the primary display.